production/helm/loki/templates/_helpers.tpl

```diff
@@ -30,6 +30,13 @@ singleBinary fullname
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return true if deployment mode is SingleBinary
+*/}}
+{{- define "loki.deployment.isSingleBinary" -}}
+  {{- eq .Values.deploymentMode "SingleBinary" }}
```

Comment:
there's also the SingleBianry<->OtherModes transition, could you add them on the check?

Reply:
sorry @DylanGuedes i didnt get the notification from your update.
Do you think the new config will work correctly?